api/src/routing.js

```diff
@@ -78,17 +82,27 @@ const handleJsonRequest = (method, path, callback) => {
     }
   });
 };
+
+const handleJsonOrCsvRequest = (method, path, callback) => {
```

Comment:
This branching seems like something the router could do for us, rather than us needed to have a route that does it. Can these (csv and json) be separated?